src/familybot/plugins/token_sender.py

```
@@ -74,65 +74,23 @@ async def _send_admin_dm(self, message: str) -> None:
-                    except Exception as e:
-                        logger.warning(f"Could not apply storage state cookies: {e}")
-            else:
-                logger.info("Using default browser profile")
-                browser = await p.chromium.launch(headless=True)
-                context = await browser.new_context()
-                page = await context.new_page()
+    async def _get_token_with_camoufox(self) -> str:
+        """Extract Steam webapi_token using Camoufox."""
+        logger.info("Starting token extraction using Camoufox...")
+
+        if BROWSER_PROFILE_PATH and os.path.exists(BROWSER_PROFILE_PATH):
```

Ruff (0.15.9)

[warning] Async functions should not use os.path methods, use trio.Path or anyio.path

(ASYNC240)